### DISCUSSION TITLE
自定义host请求头

### DISCUSSION THREAD

**Discussion by lovestudylalala** [apache/apisix#5505] [Q&A]
请问下自定义host请求头是在哪个版本开始废弃了这项功能呢，感觉这个功能在实际应用中还挺实用的

**Comment by tokers** [comment]
> 请问下自定义host请求头是在哪个版本开始废弃了这项功能呢，感觉这个功能在实际应用中还挺实用的

Do you mean modifying HTTP request headers?

**Reply by lovestudylalala** [reply]
![image](https://user-images.githubusercontent.com/55313374/141752404-4a1eb426-5d95-4b0e-9320-c5c39de3fefb.png)

**Reply by juzhiyuan** [reply]
We had a discussion on this in March 2021. see https://lists.apache.org/thread/h30qwxolw6wojq3f9m32nprnzmdrfx5z

**Reply by lovestudylalala** [reply]
thanks
